src/constants.nim

```diff
@@ -81,10 +81,11 @@ proc ensureDirectories*(dataDir, tmpDir, logDir: string) =
 
 # This is changed during compilation by reading the VERSION file
 const DESKTOP_VERSION {.strdefine.} = "0.0.0"
+const STATUSGO_VERSION* {.strdefine.} = "0.0.0"
```

Comment:
It seems to work, but I don't get how. I tried searching for `STATUSGO_VERSION` and I can't see it. Did you forget to push a Makefile change?

Reply:
Ouch :D of course!

Reply:
Then I wonder _HOW_ is this working for you...

Reply:
> Then I wonder _HOW_ is this working for you...

I have not tested. I just meant that it seems to work in your screenshots

Reply:
Right; I pushed the missing Makefile changes